dockers/docker-base/Dockerfile.j2

```diff
@@ -23,6 +23,10 @@ RUN rm -rf               \
     /var/cache/man/*     \
     /usr/share/locale/*
 
+{% if APT_CACHER_NG_PROXY.strip() -%}
```

Comment:
>APT_CACHER_NG_PROXY [](start = 6, length = 19)

Suggest rename to APT_PROXY, Apt-Cacher-NG is a specific apt proxy implementation.